airflow/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6@sha256:c5ffdf5938d73283cec018f2adf59f0ed9f8c376d93e415a27b16c3c6aad6f45 AS airflow-build-image
```

Comment:
Should we move vector out of the sandbox now that we have "top-level" images depending on it?

Reply:
Yes, this was on my TODO list before merging. Usually I add this TODO list to the description of the pull request but I forgot it this time.